documents/src/pages/elements/loader.md

```diff
@@ -47,3 +47,16 @@ Add `ef-loader` into your document, and the animation will begin. When the loade
 ```html
 <ef-loader></ef-loader>
 ```
+
+## Accessibility
+::a11y-intro::
+
+`ef-loader` is assigned `role="progressbar"` and can include `aria-label` with `aria-live` to announce loading and loaded states.
```

Comment:
I think accessibility section needs should reference this document: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role and just provide one or two paragraphs with some key points.

Reply:
Thanks Aleks, updated.